common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1176,14 +1177,14 @@
     "figura.docs.model_part.get_secondary_texture": "Gets the secondary texture of this part\nReturns two values, first being the override type, second being the value, if any.",
     "figura.docs.model_part.get_secondary_defined_textures": "Gets the secondary textures of this part\nReturns a table of each texture for the specified face.",
     "figura.docs.model_part.set_secondary_texture": "Sets the secondary texture override of this part\nCheck the TextureType types in the list docs\nIf using \"resource\", the second parameter should indicate the path to the Minecraft texture you want to use\nIf using \"custom\", the second parameter should indicate a texture object",
-    "figura.docs.model_part.get_textures": "Returns a table with all textures used by this part\nDo not include children textures, so groups usually will return an empty table",
-    "figura.docs.model_part.part_to_world_matrix": "Gets a matrix which transforms a point from this part's position to a world location\nRecommended to use this in POST_RENDER, as by then the matrix is updated\nIn RENDER it will be 1 frame behind the part's visual position for that frame\nAlso, if the model is not rendered in-world, the part's matrix will not be updated\nPaperdoll rendering and other UI rendering will not affect this matrix",
+    "figura.docs.model_part.get_textures": "Returns a table with all textures used by this part\nDoes not include children textures, so groups usually will return an empty table",
+    "figura.docs.model_part.part_to_world_matrix": "Gets a matrix which transforms a point from this part's position to a world location\nRecommended to use this in POST_RENDER, as by then the matrix is updated\nIn RENDER it will be 1 frame behind the part's visual position for that frame\nAlso, if the model is not rendered in-world, the part's matrix will not be updated\nPaperdoll rendering and other UI renderings will not affect this matrix",
```

Tbh not a fan of "renderings" but that might just be me.
```suggestion
    "figura.docs.model_part.part_to_world_matrix": "Gets a matrix which transforms a point from this part's position to a world location\nRecommended to use this in POST_RENDER, as by then the matrix is updated\nIn RENDER it will be 1 frame behind the part's visual position for that frame\nAlso, if the model is not rendered in-world, the part's matrix will not be updated\nPaperdoll rendering and other UI rendering will not affect this matrix",
```

---

I think the sentence should be rewritten if we do not go with "renderings"

---

I wouldn't know where to start tbh. The reason I went with "rendering" is because it seems natural to treat all rendering for UIs as a single unit, I'm not the Java programmer though so I have no idea what I'm talking about lmao.

---

rendering is not a single unit, leave it as renderings, even if it's not an actual word it's what makes most sense